src/test/java/tests/LoginTest.java

```diff
@@ -88,4 +91,39 @@ public void tc05_SystemShowsMessageWhenUserEntersWrongPasswordSeveralTimes() {
         LogHelper.info("Verify that system shows message when user enters wrong password several times");
         Assert.assertEquals(actual, errorMsg, "An error message is not displayed as expected result");
     }
+
+    @Test(description = "User is redirected to Home page after logging out")
+    public void tc06_UserIsRedirectedToHomePageAfterLoggingOut() {
+        LogHelper.info("Click login tab");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Enter valid data and click Login button");
+        loginPage.login(Constant.USERNAME, Constant.PASSWORD);
+
+        LogHelper.info("Get value of boolean function to know Logout tab is displayed");
+        boolean logoutTabAppearance = loginPage.isLogoutTabDisplayed();
+
+        LogHelper.info("Click contact tab");
+        ContactPage contactPage = new ContactPage();
+        contactPage.clickContactTab();
+
+        LogHelper.info("Click logout tab");
+        loginPage.clickLogoutTab();
+
+        LogHelper.info("Get value of boolean function to know Login tab is displayed");
+        boolean loginTabAppearance = loginPage.isLoginTabDisplayed();
+
+        LogHelper.info("Verify that Logout tab is displayed after logging in successfully.");
+        Assert.assertTrue(logoutTabAppearance, "Logout tab is not displayed as expected");
```

Comment:
this `"Verify that Logout tab is displayed after logging in successfully."` step should locate right after `"Get value of boolean function to know Logout tab is displayed"` step

Reply:
Fixed, please check again.